docs/faq.md

```diff
@@ -93,6 +93,16 @@ If your fuzz target is running for many days and does not find bugs or new cover
 
 In either case, look at the [coverage reports](clusterfuzz.md#coverage-reports) for your target(s) and figure out why some parts of the code are not covered. 
 
+## Why are code coverage reports public?
+
+We work with open source projects and try to keep as much information public as
+possible. The only information that we keep private is the following:
+- Bugs that were found in the past 90 days and were not fixed 30+ days ago.
```

Comment:
nit: this is a little confusing, we can just point to our disclosure policies (https://github.com/google/oss-fuzz#bug-disclosure-guidelines)

Maybe just replace with:

"Initially reported bugs. See disclosure policy."

Reply:
Thanks, I've removed that unnecessary enumeration as per your suggestion.